tool-plugins/intellij/src/main/java/org/ballerinalang/plugins/idea/codeinsight/daemon/impl/BallerinaAnnotator.java

```diff
@@ -211,11 +212,33 @@ private void annotateLeafPsiElementNodes(@NotNull PsiElement element, @NotNull A
             annotateStringLiteralTemplateEnd(element, holder);
         } else if (elementType == BallerinaTypes.DOCUMENTATION_TEMPLATE_ATTRIBUTE_START) {
             // Doc type.
+            String msg = null;
+            switch (element.getText().charAt(0)) {
+                case 'T':
+                    msg = "Receiver";
```

Comment:
Isn't it more suitable to represent these strings Eg:- "Receiver" constants?

Reply:
We don't need them anywhere else at the moment and this implementation will be changed in the plugin rewrite soon. So we can have these as constants in the new plugin if needed.